tests/driver_api.c

```diff
@@ -0,0 +1,173 @@
+#include <time.h>
```

Comment:
How about changing file name to `tests/driver.c` simply? A file containing `api` implies its offering of certain request/response pair. However, it does not really behave as API to me.